app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -147,7 +193,16 @@ function fetchNextFollowingPage(el) {
       '/followings/organizations',
       insertNext({ elId: 'follows', action }, buildFollowsHTML),
     );
+  } else if (action.includes('hidden_tags')) {
+    fetchNext(
+      el,
+      '/followings/tags',
+      insertNext({ elId: 'follows', action }, buildTagsHTML),
+    );
```

Comment:
I'm not very happy with this change either, but again, it would require _a lot of_ refactoring to change the way these statements are being checked and should be part of a bigger refactor. I could have let this be a part of the `else` statement but I don't think we should have a catch-all else. Note the comment two lines down.

Reply:
Also it would be nice to change the param name here (and in the view) to `controller_action` as well, but it's not clear what else that might affect :(

Reply:
Yip exactly! We want to rename the variable from the start. We could potentially rename it from here already but it definitely affects just too many views if we rename it from the view level. I'm going to capture these refactor notes and add it to an issue so that we can do it properly and with tests in place.